gcc/config/sol2-c.c

```diff
@@ -60,14 +62,15 @@ static const format_char_info bitfield_string_type =
 
 static const format_char_info cmn_err_char_table[] =
 {
+  /*                     none     hh       h        l        ll       L        z        t        j        H       D       DD */
   /* C89 conversion specifiers.  */
```

Comment:
Did this table go wrong sometime before, or am I missing why it grows 3 extra feels for two extra formats?
(I'm guessing the latter?)

Reply:
It was not updated when the commit below went into gcc, growing the list of formats. BADLEN is `{0, NULL, NULL}` so it did not matter, but I expanded it to match while I was in here.

```
commit 9a8ce21f3b47a109b9ac35239e21198bb36e1146
Author: Jon Grimm <jgrimm2@us.ibm.com>
Date:   Fri Dec 16 06:38:26 2005 +0000
            * c-format.h (enum format_lengths): Add new enumeration values
            FMT_LEN_H, FMT_LEN_D and FMT_LEN_DD.
            * c-format.c (printf_length_specs, scanf_length_specs): Add
            entries for H, D, DD.
```